rotkehlchen/tests/unit/accounting/test_default_settings.py

```diff
@@ -0,0 +1,332 @@
+    (HistoryEventType.RECEIVE, HistoryEventSubType.AIRDROP, False, False),
+    (HistoryEventType.RECEIVE, HistoryEventSubType.AIRDROP, True, True),
+    (HistoryEventType.RECEIVE, HistoryEventSubType.REWARD, True, False),
+])
+@pytest.mark.parametrize('mocked_price_queries', [prices])
+def test_accounting_receive_settings(
+        accounting_pot: 'AccountingPot',
+        event_type: 'HistoryEventType',
+        event_subtype: 'HistoryEventSubType',
+        is_taxable: bool,
+):
+    """Test that the default accounting settings for receiving are correct"""
+    _gain_one_ether(
+        transactions=accounting_pot.transactions,
+        event_type=event_type,
+        event_subtype=event_subtype,
+    )
+
+    expected_event = ProcessedAccountingEvent(
+        type=AccountingEventType.TRANSACTION_EVENT,
+        notes='Received 1 ETH',
+        location=Location.BLOCKCHAIN,
+        timestamp=TIMESTAMP_1_SECS,
+        asset=A_ETH,
+        free_amount=ZERO if is_taxable else ONE,
+        taxable_amount=ONE if is_taxable else ZERO,
+        price=Price(FVal(1862.06)),
+        pnl=PNL(
+            taxable=FVal(1862.06) if is_taxable else ZERO,
+            free=ZERO,
+        ),
+        cost_basis=None,
+        index=0,
+        extra_data={'tx_hash': EXAMPLE_TX_HASH_HEX},
+    )
+    expected_event.count_entire_amount_spend = False
+    expected_event.count_cost_basis_pnl = is_taxable
+
+    assert accounting_pot.processed_events == [expected_event]
+    assert accounting_pot.pnls.taxable == FVal(1862.06) if is_taxable else ZERO
+    assert accounting_pot.pnls.free == ZERO
+
+
+@pytest.mark.parametrize('event_type,event_subtype,is_taxable,counterparty,gas_taxable', [
+    (HistoryEventType.SPEND, HistoryEventSubType.NONE, True, None, False),
+    (HistoryEventType.SPEND, HistoryEventSubType.FEE, True, None, False),
+    (HistoryEventType.SPEND, HistoryEventSubType.FEE, True, CPT_GAS, True),
+    (HistoryEventType.SPEND, HistoryEventSubType.FEE, False, CPT_GAS, False),
+    (HistoryEventType.DEPOSIT, HistoryEventSubType.NONE, False, None, False),
+    (HistoryEventType.RENEW, HistoryEventSubType.NONE, True, None, False),
+])
+@pytest.mark.parametrize('mocked_price_queries', [prices])
+def test_accounting_spend_settings(
+        accounting_pot: 'AccountingPot',
+        event_type: 'HistoryEventType',
+        event_subtype: 'HistoryEventSubType',
+        is_taxable: bool,
+        counterparty: Optional[str],
+):
+    # Gain some ETH so that the spending event has something to spend
+    _gain_one_ether(transactions=accounting_pot.transactions)
+    spend_event = HistoryBaseEntry(
+        event_identifier=EXAMPLE_EVM_HASH,
+        sequence_index=0,
+        timestamp=TIMESTAMP_2_MS,
+        location=Location.BLOCKCHAIN,
+        location_label=EXAMPLE_ADDRESS,
+        asset=A_ETH,
+        balance=Balance(amount=FVal(0.5)),
+        notes='Send 1 ETH to 0xABC',
```

Comment:
You are sneind half an ETH, no?

```suggestion
        notes='Send 0.5 ETH to 0xABC',
```